### ISSUE TITLE
Docs page has only "Extensibility Options"

### ISSUE THREAD

**Issue by mightyiam** [interlockjs/www.interlockjs.com#13]
![screenshot_2016-10-29_01-42-53](https://cloud.githubusercontent.com/assets/635591/19824541/1844f400-9d79-11e6-95dc-ed17b4b2d496.png)

Is this the actual current content there?

**Comment by mightyiam** [comment]
The only link I could find to the build docs was at the bottom of http://interlockjs.com/docs/extensibility.

**Comment by divmain** [comment]
Yeah, I think I mentioned this in Gitter.  I don't yet know what the extent of the docs will look like yet, how navigation should work between it all, or what will be contained on this `docs/` landing page.  What's there is definitely a place-holder.

Do you have any thoughts?  At the moment, I'm thinking about linking to sub-sections only, with short descriptions so that people understand why its relevant and what they'd want it for.  Then deal with sub-section navigation at a lower level.

There's also the site-wide instant search.  That's great for finding content you know is there, or think should be there somewhere.  But it won't provide discoverability.
